nionswift_plugin/usim/InstrumentDevice.py

```diff
@@ -295,14 +351,13 @@ def __init__(self, instrument_id: str):
         self.property_changed_event = Event.Event()
         self.__camera_frame_event = threading.Event()
         self.__samples = [SampleSimulator.RectangleFlakeSample(), SampleSimulator.AmorphousSample()]
-        self.__sample_index = 0
+        self.__sample_index = 1
 
         # define the STEM geometry limits
-        self.stage_size_nm = 150
+        self.stage_size_nm = 1000
```

Comment:
Likewise - is there a reason to change this? For flake/scan, having it be 150nm is for consistency.

Reply:
I changed this so that I can run tuning. With the old stage size, the maximum angle on the camera was about 10 mrad, which is already too small to run 3rd order tuning. When you then add an aperture it makes things even worse. With the new stage size I get 100 mrad on the camera and the default aperture size is 40 mrad, which is a standard size we use on the real microsope (i.e. good settings for tuning tests). Changing this when the simulator is already loaded is a bit of a hack, so it would be great to keep it like this.
I'd rather change the scan simulator to be consistent with the new size. And for the flake sample: we can change that too to be consistent, but overall it is pretty useless anyway from my point-of-view.